pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -413,10 +413,12 @@ func (nim *nodeInfoManager) InitializeCSINodeWithAnnotation() error {
 
 func (nim *nodeInfoManager) tryInitializeCSINodeWithAnnotation(csiKubeClient clientset.Interface) error {
 	nodeInfo, err := csiKubeClient.StorageV1beta1().CSINodes().Get(string(nim.nodeName), metav1.GetOptions{})
-	if nodeInfo == nil || errors.IsNotFound(err) {
+	if errors.IsNotFound(err) {
```

Comment:
think you don't need to remove the `nodeInfo == nil` part from this check, in case there was no error but the `nodeInfo` was nil.. I'm not sure this is actually possible but the check is valid.

Reply:
fixed,